admin-dev/themes/new-theme/js/pages/invoices/index.js

```diff
@@ -0,0 +1,34 @@
+import TermsAndConditionsOptionHandler from "../order-preferences/terms-and-conditions-option-handler";
```

Comment:
is it needed? if so, shouldnt it be below license?

Reply:
oops